### DISCUSSION TITLE
Input multiple values

### DISCUSSION THREAD

**Discussion by taltshaked** [ant-design-blazor/ant-design-blazor#2396] [Q&A] [answered]
Hi,
I need a way to input multiple values from the user (numeric) in the same input box (separated by spaces). The only option I've found is to use the Select component with multiple selection mode, but since the values are unknown, I can't provide the required options to the select component.
Is there a way to achieve what I'm looking for?

**Comment by ElderJames** [comment, marked as answer]
Please try to use Select component with tag mode.
![image](https://user-images.githubusercontent.com/7550366/164725890-d37bb74c-e11d-460b-8f4e-d99e087d6f42.png)

And setup the space as the `TokenSeparators`
https://antblazor.com/en-US/components/select#components-select-demo-automatic-tokenization

**Reply by taltshaked** [reply]
Thank you.
This works but my bound values are integers so if the user types/pastes a non-numeric value, this will crash the page.
Is there a way around this?

**Reply by ElderJames** [reply]
Please create an issue for that.
